src/builder.js

```diff
@@ -1,43 +1,61 @@
-var _ = require('lodash');
-var xmlBuilder = require('xmlbuilder');
+// @ts-check
```

Comment:
Would it be possible to run a type check for all files with `@ts-check` on CI somehow? It would be neat.

Reply:
I think this can be done with typescript, but at that point may as well convert it all to typescript anyway as I don't think there's limits to the types we can define in JSDOC